packages/muelu/src/Transfers/BlockedTransfers/MueLu_BlockedCoarseMapFactory_decl.hpp

```diff
@@ -64,60 +58,77 @@
+
+    ## Input/output ##
+
+    ### User parameters for this factory ###
+    Parameter | type | default | master.xml | validated | requested | description
+    ----------|------|---------|:----------:|:---------:|:---------:|------------
+    Aggregates | Factory | null | | * | * | Generating factory for aggregates. We basically need only the number of aggregates in this factory.
+    Nullspace  | Factory | null |   | * | * | Generating factory of the fine nullspace vectors (of type "MultiVector"). In the default case the same instance of the TentativePFactory is also the generating factory for the null space vectors (on the next coarser levels). Therefore, it is recommended to declare the TentativePFactory to be the generating factory of the "Nullspace" variable globally using the FactoryManager object! For defining the near null space vectors on the finest level one should use the NullspaceFactory.
+    Striding info | std::string | {} |  | * | | String containing the striding information, e.g. {2,1} for a fixed block size of 3 with sub blocks of size 2 and 1 in each node.
+    Strided block id | LocalOrdinal | -1 |  | * | | Strided block id in strided information vector or -1 (=default) for the full block information.
+    Coarse Map | Factory | null | | * | | Generating factory of another coarse to be used for calculating the GID offset for the map to be generated.
+
+    The * in the @c master.xml column denotes that the parameter is defined in the @c master.xml file.<br>
+    The * in the @c validated column means that the parameter is declared in the list of valid input parameters (see @c GetValidParameters ).<br>
+    The * in the @c requested column states that the data is requested as input with all dependencies (see @c DeclareInput ).
+
+    ### Variables provided by this factory ###
+
+    After @c Build(), the following data is available (if requested)
 
-      overloads CoarseMapFactory. Uses a CoarseMapFactory as input parameter in the constructor and
-      automatically calculates the domain offset using the max gid from the given coarse map.
+    Parameter | generated by | description
+    ----------|--------------|------------
+    | CoarseMap | BlockedCoarseMapFactory | Map containing the coarse map used as domain map in the tentative prolongation operator
 
-     @ingroup MueLuTransferClasses
  */
 
 template <class Scalar = DefaultScalar,
           class LocalOrdinal = DefaultLocalOrdinal,
           class GlobalOrdinal = DefaultGlobalOrdinal,
           class Node = DefaultNode>
-class BlockedCoarseMapFactory : public MueLu::CoarseMapFactory<Scalar, LocalOrdinal, GlobalOrdinal, Node> { //SingleLevelFactoryBase {
+class BlockedCoarseMapFactory : public MueLu::CoarseMapFactory<Scalar, LocalOrdinal, GlobalOrdinal, Node>
+{
 #undef MUELU_BLOCKEDCOARSEMAPFACTORY_SHORT
 #include "MueLu_UseShortNames.hpp"
 
 public:
 
-  //! @name Constructors/Destructors.
-  //@{
-
-  //! Constructor
-  BlockedCoarseMapFactory();
```

Comment:
So do you want to remove all constructor/destructors?
I could understand the constructor removal if you only to construct these with Build factories but I feel keeping the destructor around is a good idea...
At this point the class will simply define the same default constructors implicitly, I'm not sure why we should remove the explicit ones.

Reply:
This was just about removing the default con-/destructor, that is auto-generated anyway.